src/planner.cc

```diff
@@ -394,6 +405,16 @@ Planner::Impl::Impl(const MachineControlConfig *config,
     const float accel = cfg_->acceleration[i] * cfg_->steps_per_mm[i];
     max_axis_accel_[i] = accel;
   }
+#if 0
```

Comment:
There is a trick with the comma to avoid looking for `size() - 1` every time to check if we need to add a comma  ... just check if we are the first element, otherwise prepend a comma; this makes it more compact:

```c++
#if 0
  fprintf(stderr, "\nmax_accelerations:\n[");
  for (unsigned i = 0; i < max_axis_accel_.size(); ++i) {
    fprintf(stderr, "%s%f", !i ? "" : ",", max_axis_accel_[(GCodeParserAxis)i]);
  }
  fprintf(stderr, "]\n");
#endif
```

(similar in other places in this file)